toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -483,45 +490,58 @@ Example: `noatime,nodiratime`
 
 ## verity type
 
-Specifies the configuration for dm-verity root integrity verification.
+Specifies the configuration for dm-verity integrity verification.
 
-- `dataPartition`: A partition configured with dm-verity, which verifies integrity
-  at each system boot.
+Note: Currently only root partition (`/`) is supported. Support for other partitions
+(e.g. `/usr`) may be added in the future.
```

Comment:
I think there's no need to add possible future additions?

Reply:
Verity is now a list. So, this is me trying to explain why you can only add a single item to the list, at the moment.

Also, there is a good chance we will support `/usr`-only verity. That is, only the `/usr` directory is protected by verity. This provides an image with some verity protection but is far easier to use than root verity (which makes `/etc` read-only).